src/Treeview.ts

```diff
@@ -38,7 +38,13 @@ const resourceIcons: [string, string][] = [
     ["Microsoft.Compute/virtualMachines/extensions", "extensions.svg"],
     ["Microsoft.Network/networkSecurityGroups", "nsg.svg"],
     ["Microsoft.Network/networkInterfaces", "nic.svg"],
-    ["Microsoft.Network/publicIPAddresses", "publicip.svg"]
+    ["Microsoft.Network/publicIPAddresses", "publicip.svg"],
+    ["Microsoft.DocumentDB/databaseAccounts", "documentdb.svg"],
```

Comment:
The documentdb icon doesn't look great in dark theme. In the Cosmos DB extension we use this one: https://github.com/Microsoft/vscode-cosmosdb/blob/master/resources/icons/dark/CosmosDBAccount.svg

That being said - the current code only supports one icon per resource. Are you interested in refactoring to support themed icons? If not - I would prefer to wait on this particular icon until we fix that. If so - we can help you along the way. VS Code lets us specify the icon as either a string or an object with the path for both "light" and "dark". See here for more info: https://github.com/Microsoft/vscode/blob/master/src/vs/vscode.d.ts#L6554

Reply:
After you asked for the screenshots I looked up how to query for dark/light themes. I’ll take a look and make the appropriate changes.